src/sql/orderbook/barn_order_rewards.sql

```diff
@@ -72,7 +72,7 @@ order_surplus AS (
         ON o.uid = oq.order_uid
     WHERE
         ss.block_deadline >= {{start_block}}
-        AND ss.block_deadline <= {{end_block}}
+        AND ss.block_deadline <= {{end_block}} + 100
```

Comment:
Better add some comment here.
```suggestion
        -- since this table filtered on block_deadline is joined with another table filtered on block_number
        -- the bound for this table need to be a bit looser.
        AND ss.block_deadline <= {{end_block}} + 100
```